lib/solr.ts

```diff
@@ -591,20 +584,46 @@ export class Client {
       })
       .join('/');
 
-    const params = {
+    const requestOptions: RequestOptions = {
       host: this.options.host,
       port: this.options.port,
-      fullPath: fullPath,
-      json: json,
-      secure: this.options.secure,
-      bigint: this.options.bigint,
-      authorization: this.options.authorization,
-      agent: this.options.agent,
-      ipVersion: this.options.ipVersion,
-      // TODO this doesn't seem to be used
-      //request: this.request,
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json; charset=utf-8',
+        'content-length': Buffer.byteLength(json),
+        accept: 'application/json; charset=utf-8',
+      },
+      path: fullPath,
```

Comment:
we lose fullPath param here and replace it with path. Which one Request expects?

Reply:
We don't lose anything. It's the exact same thing as we had before: https://github.com/lbdremy/solr-node-client/pull/278/files/3418634b61cb16b6dca20dad0a8f9d469efa5503#diff-b0d5c84599fa1c42597f5bfad83e01676fdbe2cb87068ed1f57112293b54bd67L108

Reply:
Also, fullPath is just the name of a variable. You could rename it to `path` or even `foo` if you wanted. See https://github.com/lbdremy/solr-node-client/pull/278/files/3418634b61cb16b6dca20dad0a8f9d469efa5503#diff-3b89c2e1c7e4e714474d6a2f17b55814a790482f4ef375d4f84d4895554c20d9L581